src/common/internal/test_group.ts

```diff
@@ -198,6 +220,12 @@ class TestBuilder {
     return this;
   }
 
+  before(fn: BeforeFn<FixtureSharedState, {}>): this {
```

Comment:
I used "beforeSubcases" ?

Reply:
that's kind of ambiguous between "before each subcase" and "before all subcases". Maybe "beforeEachCase"?

(Wish I'd come up with a better name than "case"/"subcase" because when we use the word "case" it always sounds ambiguous to me. But I think it would be fine.)

Reply:
Or `initSharedState`?

Reply:
Hm - ok I'll do `beforeAllSubcases` then